src/aspire/operators/polar_ft.py

```diff
@@ -116,15 +99,33 @@ def _evaluate_t(self, x):
             Fourier grid. This is an array of vectors whose first dimension
             corresponds to `x.shape[0]`, and last dimension equals `self.count`.
         """
-        nimgs = x.shape[0]
-
-        half_size = self.ntheta // 2
-
-        pf = nufft(x, self.freqs)
-
-        pf = pf.reshape((nimgs, self.nrad, half_size))
-        v = np.concatenate((pf, pf.conj()), axis=1)
-
-        # return v coefficients with the last dimension size of self.count
-        v = v.reshape(nimgs, -1)
-        return v
+        if x.dtype != self.dtype:
+            logger.warning(
```

Comment:
Nothing complains. This check and the one on line 108 were previously being handled by `Basis` when this was a subclass.

I'll change the mention of `evaluate_t` since this is now called `transform`.

Reply:
can probably raise then. This code might allow mixed modes, yuk.

Reply:
Raising now.